src/braket/experimental/autoqasm/converters/assignments.py

```diff
@@ -37,6 +37,14 @@ def visit_Assign(self, node: ast.stmt) -> ast.stmt:
         template = """
         tar_ = ag__.assign_stmt(tar_name_, val_)
         """
+        try:
+            # Assignments for main function return statements have already been handled,
+            # so return early
+            if node.value.func.attr == "assign_for_output":
```

Comment:
this `"assign_for_output"` needs to match the string inside the `ReturnTransformer` template, correct? Is it worth making this string a constant variable that can be referenced in those two places, just so that dependency is unambiguous?

Reply:
Definitely, thank you! I don't love this solution but that makes it a bit better

Reply:
Resolved with `func.__name__` so that all three locations have an explicit dependency and no checks against string literals